src/middleware/cors.ts

```
@@ -0,0 +1,22 @@
+import { defineMiddleware } from "astro:middleware";
+
+export const corsMiddleware = defineMiddleware(async ({ request }, next) => {
+    console.info('corsMiddleware');
+    const origin = request.headers.get('Origin');
+
+    if (request.method === 'OPTIONS') {
+        let headers = new Headers();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'headers' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)